docs/concepts/flows.md

```diff
@@ -61,6 +61,7 @@ query: |
 
 By decorating the `go.func_decl` fact with `@review.comment` the review Flow's comment Function extracts the file and line from the query result, to be used by the review Flow to comment on that line.
 
+<!-- TODO fix HTML link. -->
```

Comment:
Where does it point to now? Is there a quick hack we can do?

Reply:
Looks like it's not broken on the deployed site. Do you mean it would just be nicer to have this in markdown?

Reply:
MkDocs ignores raw HTML. The hack is just to link to the full url of tenets (https://www.codelingo.io/docs/concepts/tenets/) I'll do that...

Reply:
oh.. I'm getting 404 on localhost. I'll remove the todo.